carbon-design-system/carbon
# feat(Layout): add container sizing tokens

closed #8552 

## Changes
Adds container sizing tokens:
$size-xs,
$size-sm,
$size-md
$size-lg,
$size-xl,
$size-2xl

## Testing
- run `yarn build` inside @carbon/layout 
- check that `/layout/scss/generated/_size.scss` was created properly
